wsgi.py

```diff
@@ -16,13 +16,15 @@
 import os
 import sys
 from os.path import abspath, join
+import subprocess
 
 CURR_DIR = os.path.dirname(__file__)
 sys.path.append(os.path.join(CURR_DIR, 'src'))
 sys.path.append(abspath(join(CURR_DIR, 'libs', 'django-rest-framework-0.4')))
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "project.settings")
 
+subprocess.call("python src/manage.py collectstatic --noinput", shell=True)
```

Comment:
So `collectstatic` runs every time the server process is restarted? Maybe not ideal but definitely won't hurt. I like the creativity!